src/main/java/bedwars/TeamInfo.java

```diff
@@ -63,4 +58,51 @@ public void stopGenerators() {
     emerald.stop();
   }
 
+  public String getName() {
```

Comment:
Freaking getters and setters mannnnn
fuck encapsulation

Reply:
no